open-company/open-company-web
# Nux tooltips tweaks

Card: https://trello.com/c/KFi6hy8r

To test:
- signup
- [x] do you see the first tooltip to create your first post? Good
- click the green button
- [x] do you see the tooltip inside cmail? Good
- switch browser to mobile size
- refresh
- [x] do you see only the first part of the tooltip on mobile? Good
- switch back to desktop size
- refresh
- add a post and publish it
- [x] do you see a floating tooltip pointing at WST of your newly published post? Good
- switch to mobile and refresh
- [x] do you NOT see it? Good
- switch to desktop and refresh
- [x] can you dismiss it? Good
